src/battle_animation.cpp

```diff
@@ -31,8 +31,8 @@
 #include "player.h"
```

Comment:
> 2.11: RM2000: When an enemy uses a technique/ability, the animation,

Approved.
Only this commit message... 6. is e.g. misleading now

In general it is not helpful.
I would completely wipe it and simply mention that skills on allies play the sound of the animation instead of the attack sound.